Source/Yaml/_main_.env

```diff
@@ -181,3 +206,40 @@ _t4dSrcYamlUpdateLine(){
     fi
 ###############################
 }
+
+_t4dSrcYamlSearch(){
+# Generated From $Tools4Dev_PATH/Engine/G4d/templates.env
+# v1.8.0
+###### _t4dSrcYamlSearch
+#   - Name:
+#       _t4dSrcYamlSearch
+###
+#   - Synopsis:
+#       search parsed content to show human readable output
+###
+#   - Usage:
+#        _t4dSrcYamlSearch <File> <ymlArg>
+###
+#   - Note:
+#       <Required>
+#       [Optionnal]
+#       {XXXXXXX}       = Value To Be Changed, if {URL} then replace {URL} with a valid URL
+###
+###### DOCUMENTATION BLOC #####
+########### CODE BLOC #########
+    local _t4dErrorCode=1
+    local _File="$1"
+    local _YmlArg="$2"
+
+    cat $_File | grep "$_YmlArg" > "$_File.tmp"
```

Comment:
you should never use argument without adding quotes, it's a really bad way to do shell, even more with cat